### DISCUSSION TITLE
Please Refund

### DISCUSSION THREAD

**Discussion by smhylc** [aclap-dev/video-downloadhelper#382] [Help and Q&A]
### Your question

Hello, I made a plugin purchase (3-4 hours ago), but when 2 videos are opened at the same time in the brave browser, the plugin cancels other video downloads. 

This plugin does not meet my request and I request license cancellation and refund for this plugin, which I purchased 3-4 hours ago using my customer rights. 

mail : semih.yolcu01@gmail.com 

### Addon & CoApp version and details

version: 8.2.2.9
channel: stable
build date: 2024-03-19
build options: linuxlic: false, noyt: true, browser: chrome
lang: tr
license: accepted
key (only 16 first characters): e22fbd7c67ca4886
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/123.0.0.0 Safari/537.36
coapp: yes. Info:
{
  "id": "net.downloadhelper.coapp",
  "name": "VdhCoApp",
  "version": "2.0.19",
  "binary": "C:\\Program Files\\DownloadHelper CoApp\\vdhcoapp.exe",
  "displayName": "VdhCoApp",
  "description": "Video DownloadHelper companion app",
  "target": {
    "os": "windows",
    "arch": "x86_64",
    "node": "18"
  },
  "converterBinary": "C:\\Program Files\\DownloadHelper CoApp\\ffmpeg.exe",
  "converterBase": "ffmpeg",
  "converterBaseVersion": "c30f360955"
}

**Comment by smhylc** [comment]
Other videos disappear in the video download queue or are downloaded incompletely. I would like to return the program.

**Reply by jcfrog** [reply]
hi,
I sent you an email.
